docs/Wdio_Utility.md

```diff
@@ -0,0 +1,125 @@
+const webpackConfig = require('./webpack.config.js');
+const WebPackDevService = require('terra-toolkit/lib/wdio/services').WebPackDevService;
+const localIP = require('ip');
+
+const port = 8080;
+
+let specs;
+const isRepoTest = !process.cwd().includes('/packages/');
+
+if (isRepoTest) {
+  specs = path.join(__dirname, 'packages', 'terra-**', 'tests', 'wdio', '**', '*-spec.js');
+} else {
+  specs = path.join('tests', 'wdio', '**', '*-spec.js');
+}
+
+const config = {
+  ...wdioConf.config,
+
+  // Point baseURL to the site to be tested
+  baseUrl: `http://${localIP.address()}:${port}`,
+  specs: [specs],
+
+  // Use Terra-toolkit's WebPackDevService
+  services: wdioConf.config.services.concat([WebPackDevService]),
+
+  // Configuration for WebPackDevService
+  webpackConfig,
+  webpackPort: port,
+};
+
+exports.config = config;
+```
+
+## Writing Tests
+
+There are a few things to note about the webdriver.io configuration provided by Terra-Toolkit:
+
+- Test files should use `*-spec.js` naming format. The default spec search path is `./tests/wdio/**/*-spec.js`.
+- Use `/test_url_path` to direct test urls. This is appended to the `baseUrl` provided in the config.
+
+Then, to assist with testing, the TerraService provides the custom assertions `accessible` and `matchReference` and a Terra global helper to make testing easier:
+
+- `accessible()` validates the `axe()` accessibility assertions on the specified viewports are successful.
+- `matchReference()` validates the `checkElement` visual regression assertions on the specified viewports are successful.
+- `Terra.viewports(name)` takes the viewport key name(s) and returns an array of { height, width } objects representing the respective terra viewport size(s).
+    - Use this function to resize the browser or pass the viewport sizes to the accessibility and visual regression commands.
+    - By default returns all viewports if not name key are provided.
+
+```js
+// These globals are provide via the Terra Service
+/* global browser, describe, it, expect, viewport */
+
+describe('Basic Test', () => {
+  // Only test tiny and huge viewports
+  const viewports = Terra.viewports('tiny', 'huge');
+
+  before(() => browser.url('/test.html'));
+
+  it('checks accessibility', () => {
+    expect(browser.axe()).to.be.accessible();
+  });
+
+  it('checks visual comparison', () => {
+    const screenshots = browser.checkViewport({ viewports });
+    expect(screenshots).to.matchReference();
+  });
+
+  it('switches viewport sizes', () => {
+    viewports.forEach(size, () => {
+      browser.setViewportSize(size);
+    });
+  });
+});
+```
+
+## Running Tests
+Installation of webdriver.io, providing access to the wdio test runner. The wdio test runner requires a configuration file to be passed either from the current directory or via the `--config` or `-c` followed by the config path.
+
+```javascript
+// Script at the root-level of a mono-repo with config in same directory
```

Comment:
nit: Might notes this is an npm script. e.g. `// NPM script at ...`

and then in the example below use the script key

```
  "scripts": {
    "wdio": "wdio";
  }
```